packages/components/pie-modal/test/visual/pie-modal.spec.ts

```diff
@@ -343,46 +339,20 @@ test.describe('Prop: `leadingAction`', () => {
     });
 });
 
-test.describe('Prop: `supportingAction`', () => {
-    test.describe('when `leadingAction` prop exists', () => {
-        test('should display `supportingAction` correctly', async ({ mount, page }) => {
+test.describe('Prop: `supportingActionText`', () => {
+    test.describe('when `leadingActionText` prop exists', () => {
+        test('should display modal footer if `supportingActionText` is provided', async ({ mount, page }) => {
             await mount(PieModal, {
                 props: {
                     heading: 'This is a modal heading',
                     isOpen: true,
-                    leadingAction: {
-                        text: 'Confirm',
-                        variant: 'primary',
-                        ariaLabel: 'Confirmation text',
-                    },
-                    supportingAction: {
-                        text: 'Cancel',
-                        variant: 'ghost',
-                        ariaLabel: 'Cancellation text',
-                    },
-                } as ModalProps,
-            });
-
-            const modal = page.locator(componentSelector);
-            await expect.soft(modal).toBeVisible();
-
-            await percySnapshot(page, 'Modal displays supportingAction alongside leadingAction');
-        });
-
-        test('should display modal footer if `LeadingAction` is provided alongside `supportingAction`', async ({ mount, page }) => {
```

Comment:
This test was an exact duplicate of another, so I removed it because we already have the visual coverage.